app/controllers/samvera/persona/users_controller.rb

```diff
@@ -116,8 +194,25 @@ def app_view_path
       prepend_view_path Rails.root.join('app', 'views')
     end
 
+    def last_sign_in(user)
+      user.last_sign_in_at? ? user.last_sign_in_at : user.invitation_sent_at
+    end
+
+    def format_roles(roles)
+      formatted_roles = []
+      roles.each do |role|
+        formatted_roles.append("<li>#{role}</li>")
+      end
+      formatted_roles
+    end
+
+    def user_status(user)
+      user.accepted_or_not_invited? ? 'Active' : 'Pending'
+    end
+
     def user_params
-      params.require(:user).permit(:email, :username, :password, :password_confirmation, :is_superadmin, :facebook_handle, :twitter_handle, :googleplus_handle, :display_name, :address, :department, :title, :office, :chat_id, :website, :affiliation, :telephone, :avatar, :group_list, :linkedin_handle, :orcid, :arkivo_token, :arkivo_subscription, :zotero_token, :zotero_userid, :preferred_locale, role_ids: [])
+      new_params = params.require(:user).permit(:email, :username, :password, :password_confirmation, :is_superadmin, :facebook_handle, :twitter_handle, :googleplus_handle, :display_name, :address, :department, :title, :office, :chat_id, :website, :affiliation, :telephone, :avatar, :group_list, :linkedin_handle, :orcid, :arkivo_token, :arkivo_subscription, :zotero_token, :zotero_userid, :preferred_locale, role_ids: [])
+      new_params
```

Comment:
In my initial work on this, the sorting and searching did not work without the explicit variable assignment. As it has been changed significantly since then, I will see if rolling it back maintains functionality.